tests/sys/fs/fusefs/ioctl.cc

```diff
@@ -0,0 +1,187 @@
+/*-
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright (c) 2024 The FreeBSD Foundation
```

Comment:
As emaste said, no copyright assignment is expected.  Better to put your own name on this line.